### ISSUE TITLE
Mother 1.3.3 Root/Child Issues

### ISSUE THREAD

**Issue by pizzathehut893** [Stubs42/OrangeLine#10]
So I'm Having an issue selecting a root note or any child scale.  

It quantizes incoming cv to any scale I select, but the root is always in C. Turning the root knob or child knob doesn’t make a difference no matter if I initialize it, select random notes for the scale, or open a new patch and use it as simple as possible. The root note is always C no matter what when the incoming signal is 0V.  

After spending more time trying to figure out the problem, I realized that if I do select a different root, it randomly sets the outputted note to B no matter which root I use.  If I change it again, it's back to C.  All the while changing the child setting makes no change as usual.  

The root setting only works if I have no other notes selected (just the root note is outputted).

I come across this problem no matter how I use Mother, but as a simple example patch where I run into the issue: I use a sequencer with a range of -1V to +1V as a voltage source, which runs into the CV IN on Mother, and outputting the quantized signal into the v/oct input of any oscillator.  I'll attach some basic examples.  
![Screenshot (2)](https://user-images.githubusercontent.com/62157418/76657497-06865280-6540-11ea-82a5-457e339a5c20.png)
![Screenshot (3)](https://user-images.githubusercontent.com/62157418/76657498-07b77f80-6540-11ea-9a08-4b559acbe121.png)
![Screenshot (4)](https://user-images.githubusercontent.com/62157418/76657499-07b77f80-6540-11ea-8112-181922ab0fe1.png)

**Comment by Stubs42** [comment]
Hi,
Maybe you misunderstand the root and scale functionality. Changing root does not transpose the output but transposes the selected notes and weights. If you send a C4 in, Mother will output C4 if C4 is a valid note in the current selected scale. If not it quantizes nearest down. That's why you get a B if C is not in scale but B is.
This works similar to the transpose functionality of ML Quantum. 
Did this help?

**Comment by pizzathehut893** [comment]
Yes I understand that.  I just assumed that sending 0V into mother would automatically make mother output the selected root note.  Am I incorrect about that statement?  Shifting the transpose in  ML's Quantum will shift the root note when I send 0V into it.  You can see that the root isn't shifted at all from the screenshots I provided above.  

And as I said before, changing the child function doesn't shift the scale either, and I know it's supposed to output the same notes, but my understanding is that it shifted the root of the same notes (like changing the mode in a scale).  If I send a sequence into mother, changing the child knob doesn't make a difference in notes at all like it does in other's videos (for example Omri's).  

All together, mother always outputs the correct notes.  But if, as an example, C is in whatever scale and root I select, then 0V always = a C note.  This makes movements with the child function nonexistent.

**Comment by pizzathehut893** [comment]
As a side note, is it possible to download earlier versions of mother to see if this bug persists with them?

**Comment by Stubs42** [comment]
To answer your first question. Yes, you are incorrect about that statement. Root and Child settings do not influence the output of Mother directly. The mother scale defines which notes Mother is allowed to output. By definition, child scales share the same notes. So, if you do not use Mothers Fate and weight settings, but use Mother as a plain Quantizer, any change of the child knob or child cv will not have any effect on the cv out of Mother. The input not is always quantized nearest down to the next allowed note selected by the mother scale. So if you set up a major scale and send 0v C4 to mother, changing the root will make the output toggle between C4 and B3.
In other words root transposes the allowed notes but not the output.
Changing the child only makes sense if you use Fate and weight settings to do probabilistic quantizing Mother was made for. Please watch Omris and my own YouTube video carefully.
Your screenshots are not showing any unexpected behavour. You only send a C through Mother. C is a valid note for F major es well as for G Dorian. So Mother outputs the C. Nothing wrong with that.

**Comment by pizzathehut893** [comment]
Gotcha.  Thanks for the help.
